src/Resources/ResourceManager/Implementation/CmdletBase/ResourceManagerCmdletBase.cs

```diff
@@ -66,14 +66,6 @@ public abstract class ResourceManagerCmdletBase : AzureRMCmdlet
         /// </summary>
         private SubscriptionSdkClient subscriptionSdkClient;
 
-        /// <summary>
-        /// Gets or sets the API version.
-        /// </summary>
-        [CmdletParameterBreakingChange("ApiVersion", ChangeDescription = "Parameter is being deprecated without being replaced")]
-        [Parameter(Mandatory = false, HelpMessage = "When set, indicates the version of the resource provider API to use. If not specified, the API version is automatically determined as the latest available.")]
-        [ValidateNotNullOrEmpty]
-        public virtual string ApiVersion { get; set; }
```

Comment:
removing parameter is breaking change and is not allowed in non-breaking change release